webpack/scenes/ContentViews/Publish/CVPublishFinish.js

```diff
@@ -46,9 +49,8 @@ const CVPublishFinish = ({
     if (currentStep !== 1) {
       dispatch(stopPollingTask(cvVersionPublishKey(cvId, versionCount)));
       setCurrentStep(1);
+      setPolling(false);
       setIsOpen(false);
-      dispatch(getContentViewDetails(cvId));
-      dispatch(getContentViews);
```

Comment:
We do need these to happen to trigger a second publish without refresh. Without these trying to publish fails without refresh after a publish.